typescript/cli/src/deploy/warp.ts

```diff
@@ -313,3 +327,57 @@ async function getWarpCoreConfig(
 
   return warpCoreConfig;
 }
+
+export async function runWarpRouteApply(params: ApplyParams) {
+  const {
+    warpDeployConfig,
+    warpCoreConfig,
+    context: { registry, multiProvider },
+  } = params;
+
+  // Addresses used to get static Ism factories
+  const addresses = await registry.getAddresses();
+
+  // Convert warpCoreConfig.tokens[] into a mapping of { [chainName]: Config }
+  // This allows O(1) reads within the loop
+  const warpCoreByChain = Object.fromEntries(
+    warpCoreConfig.tokens.map((token) => [token.chainName, token]),
+  );
+
+  // Attempt to update Warp Routes
+  // Can update existing or deploy new contracts
+  logGray(`Comparing target and onchain Warp configs`);
+  await promiseObjAll(
+    objMap(warpDeployConfig, async (chain, config) => {
+      try {
+        // Update Warp
+        config.ismFactoryAddresses = addresses[
+          chain
+        ] as ProxyFactoryFactoriesAddresses;
+        const evmERC20WarpModule = new EvmERC20WarpModule(multiProvider, {
+          config,
+          chain,
+          addresses: {
+            deployedTokenRoute: warpCoreByChain[chain].addressOrDenom!,
+          },
+        });
+        const transactions = await evmERC20WarpModule.update(config);
+
+        // Send Txs
+        if (transactions.length) {
+          for (const transaction of transactions) {
+            await multiProvider.sendTransaction(chain, transaction);
+          }
+
+          logGreen(`Warp config updated on ${chain} chain.`);
+        } else {
+          logGreen(
+            `Warp config on ${chain} chain is the same as target. No updates needed.`,
+          );
+        }
+      } catch (e) {
+        logRed(`Warp config on ${chain} chain failed to update.`, e);
```

Comment:
nit, from what I see in a couple other commands:
```suggestion
          logGreen(`Warp config updated on ${chain}.`);
        } else {
          logGreen(
            `Warp config on ${chain} is the same as target. No updates needed.`,
          );
        }
      } catch (e) {
        logRed(`Warp config on ${chain} failed to update.`, e);
```

Reply:
fixed